triton/nonblocking_stream.py

```diff
@@ -10,6 +10,8 @@
 Adapted from https://github.com/rhettg/BlueOx/blob/master/blueox/network.py
 
 """
+from __future__ import unicode_literals
```

Comment:
`unicode_literals` doesn't have any scope outside of the file (e.g. if you import a string literal from another file, it will be an old-style ascii-string). Current guidelines recommend that, if you do import unicode literals, you do it in every file. This will be the case for triton (except for test files), but I'm doing separate PRs for each source file so that you don't have to read through a massive single PR.

There are a handful of literals in this file (`META_STRUCT_FMT = "!B64p64p"` and `return NonblockingStream(stream_name, s_config['partition_key'])`, for example). It's not _really_ necessary in this file, but I wanted to import it for testing to make sure that, if anything broke, it would break early.

I want all strings within triton to be unicode strings, only being converted back into older-style `str` objects when needed (for example, by the `struct` library).